Sources/ParseLiveQuery/Client.swift

```diff
@@ -23,7 +23,7 @@ public class Client: NSObject {
     let clientKey: String?
 
     var socket: SRWebSocket?
-    var userDisconnected = false
+    public var userDisconnected = false
```

Comment:
Why does this need to be made public - this only controls whether or not 'disconnect()' was explicitly called, and the person using the `Client` should know this already, no?